clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -97,8 +97,8 @@ private void trySend(final long currentTimeMs) {
             unsent.timer.update(currentTimeMs);
             if (unsent.timer.isExpired()) {
                 iterator.remove();
-                unsent.callback.ifPresent(c -> c.onFailure(new TimeoutException(
-                        "Failed to send request after " + unsent.timer.timeoutMs() + " " + "ms.")));
+                unsent.callback.onFailure(new TimeoutException(
+                    "Failed to send request after " + unsent.timer.timeoutMs() + " " + "ms."));
```

Comment:
maybe 
` "Failed to send request after " + unsent.timer.timeoutMs() + " ms."`

I'm not sure why I added an extra  `+ " "`

Reply:
Ack. Wil fix